sdks/java/core/pom.xml

```diff
@@ -429,6 +429,12 @@
     </dependency>
 
     <dependency>
+      <groupId>com.google.code.findbugs</groupId>
```

Comment:
Is it now in? I am confused by this since we use findbugs annotations all over the place prior to this PR.